pkg/ssh/forward.go

```diff
@@ -86,3 +141,44 @@ func forward(localConn net.Conn, client *ssh.Client, remoteNetwork, remoteAddr s
 	waitGroup.Add(1)
 	waitGroup.Wait()
 }
+
+func reverseForward(
+	remoteConn net.Conn,
+	client *ssh.Client,
+	localNetwork, localAddr string,
+	log log.Logger,
+) {
+	// Setup localConn (type net.Conn)
+	localConn, err := net.Dial(localNetwork, localAddr)
+	if err != nil {
+		log.Debugf("error dialing remote: %v", err)
+		return
+	}
+	defer localConn.Close()
+
+	// Copy localConn.Reader to sshConn.Writer
+	waitGroup := sync.WaitGroup{}
+	go func() {
+		defer waitGroup.Done()
+		defer localConn.Close()
+
+		_, err = io.Copy(localConn, remoteConn)
+		if err != nil {
+			log.Debugf("error copying to local: %v", err)
+		}
+	}()
+	waitGroup.Add(1)
+
+	// Copy sshConn.Reader to localConn.Writer
+	go func() {
+		defer waitGroup.Done()
+		defer remoteConn.Close()
+
+		_, err = io.Copy(remoteConn, localConn)
```

Comment:
in similar cases in `loft-sh/ssh` we were using `bicopy` - why this one is different?

Reply:
Not sure here, might want to ask @FabianKramm (the reverse code is a copy of the regular forward one more or less)